docs/features/metrics.md

```diff
@@ -17,10 +17,9 @@ metric-name:
     select statement
 ```
 
-```text
 For example, below two metrics, `null-registrations` and `distinct-registrations` are defined:
```

Comment:
I thought it would be nicer, if this text is not part of the code block:
![image](https://user-images.githubusercontent.com/52512373/101230917-48c76500-3676-11eb-882f-c00611d35b63.png)
However, I can't preview the documentation, so would rely on you, Robert, to help me out here.

Reply:
Ah the pitfalls of using something like gitbook.